src/components/graph/nodes/network-modification-node.js

```diff
@@ -90,12 +93,30 @@ const NetworkModificationNode = (props) => {
     const classes = useStyles();
 
     const currentNode = useSelector((state) => state.currentTreeNode);
+    const selectedNodeForCopy = useSelector(
+        (state) => state.selectedNodeForCopy
+    );
```

Comment:
I am wondering is it is really needed to use redux to do this, maybe we could just pass a prop to the right child

Reply:
Seems tricky to update a given node (in the treeNodes model) from network-modification-tree-pane.js, to pass this data as a prop.
I'd rather use dispatch and redux.